src/app/complex/complex-details/complex-header/download-modal/download-modal.component.ts

```diff
@@ -1,50 +1,34 @@
-import {Component, Input, OnInit} from '@angular/core';
+import {Component, computed, input, OnInit, Signal} from '@angular/core';
 import {environment} from '../../../../../environments/environment';
-import {Category} from '../../../../shared/google-analytics/types/category.enum';
 import {AnalyticsService} from '../../../../shared/google-analytics/service/analytics.service';
 
+const base = environment.complex_ws_base_url;
+
+interface Format {
+  icon: string;
+  name: string;
+  url: string;
+  disabled?: boolean;
+}
+
 @Component({
   selector: 'cp-download-modal',
   templateUrl: './download-modal.component.html',
   styleUrls: ['./download-modal.component.css']
 })
 export class DownloadModalComponent implements OnInit {
-  private _complexAC: string;
+  complexAC = input.required<string>();
+
+  formats: Signal<Format[]> = computed(() => [
+    {icon: '1', name: 'PSI-MI XML 2.5', url: `${base}/export/${this.complexAC()}?format=xml25`},
+    {icon: '1', name: 'PSI-MI XML 3.0', url: `${base}/export/${this.complexAC()}?format=xml25`},
+    {icon: 'v', name: 'ComplexTab', url: `${base}/export/${this.complexAC()}?format=tab`, disabled: true},
+    {icon: 'J', name: 'PSI-MI JSON', url: `${base}/export/${this.complexAC()}`},
+  ]);
 
   constructor(private googleAnalyticsService: AnalyticsService) {
   }
 
   ngOnInit() {
   }
-
-
-  goToComplexWSJsonFormat(): void {
-    this.googleAnalyticsService.fireDownloadResourceEvent(Category.details, 'ComplexWS');
```

Comment:
Not that it's particularly relevant, but we're losing the Google Analytics events here. I prefer using `href` in the HTML instead of window.open, but if we care about the Google analytics, we're missing that.

Also, we're missing the `_blank` property now.

Reply:
I initially wanted to make the link visible and accessible to users, but actually that makes the download not work since https://stackoverflow.com/questions/49474775/chrome-65-blocks-cross-origin-a-download-client-side-workaround-to-force-down, it creates more issue since it just opens it in a new tab if its coming from a cross origin. So I'm reverting back to a click action instead. Thanks for pointing that out!

Reply:
fixed in [eef8b69](https://github.com/Complex-Portal/complex-portal-view/pull/225/commits/eef8b69584a6e77db2a8ba2706a1a3862b7319cf)